src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/IHtmlGenerator.cs

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
-using Microsoft.AspNet.Html.Abstractions;
+using Microsoft.AspNet.Html;
```

Comment:
Think you need to rebase. Fortunately have a Coherent build with the update packages now.